runtime_scan/pkg/provider/aws/client.go

```diff
@@ -398,15 +409,31 @@ func (c *Client) getInstancesFromDescribeInstancesOutput(result *ec2.DescribeIns
 				continue
 			}
 			ret = append(ret, &InstanceImpl{
-				ec2Client: c.ec2Client,
-				id:        *instance.InstanceId,
-				region:    regionID,
+				ec2Client:        c.ec2Client,
+				id:               *instance.InstanceId,
+				region:           regionID,
+				availabilityZone: *instance.Placement.AvailabilityZone,
```

Comment:
are we sure `instance.Placement` or `instance.Placement.AvailabilityZone` are not nil?

Reply:
Actually, why do we even need AZ?

Reply:
Agreed to add a validation function of the instance object, print the invalid field and skip the instance.